src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/PropertyPageDataProducer.cs

```diff
@@ -54,10 +58,19 @@ public static IEntityValue CreatePropertyPageValue(IEntityRuntimeModel runtimeMo
 
             if (requestedProperties.Kind)
             {
-                newPropertyPage.Kind = rule.PageTemplate;
+                if (rule.PageTemplate == "debuggerParent")
+                {
+                    newPropertyPage.Kind = "generic";
+                }
+                else
+                {
+                    newPropertyPage.Kind = rule.PageTemplate;
+                }
```

Comment:
nit: can use a ternary here like other assignment statements of this type in the code.

Reply:
Done